modules/core/src/v2/coins/eth.ts

```diff
@@ -47,6 +47,30 @@ interface HopPrebuild {
   signature: string,
 }
 
+/**
+ * A recipient for ETH transactions
+ */
+interface EthRecipient extends Recipient {
+  data?: string;
+}
+
+/**
+ * Final batch transaction parameters
+ */
+interface batchParams {
+  batchRecipients: EthRecipient[];
+  batchData: Buffer;
+}
+
+/**
+ * Information needed to successfully execute Batcher contract tx
+ */
+interface batchExecutionInfo {
```

Comment:
interfaces should start with a capital letter.

Reply:
Fixed.